src/Deeson/WardenThirdPartyLibraryBundle/Managers/ThirdPartyLibraryManager.php

```diff
@@ -80,7 +80,7 @@ public function onWardenSiteUpdate(SiteUpdateEvent $event) {
     $data = $event->getData();
     $site = $event->getSite();
     $libraryData = array();
-    if (isset($data->library)) {
+    if (isset($data->library)  && $data->library !== 'disabled') {
```

Comment:
Ok! I'll push the changes. I wasn't sure if it was something that you had agreed upon, since the Drupal module appears to push 'disabled' all over the place when features are disabled. Checking for an array seems like a good solution.